pennylane/plugins/default_gaussian.py

```diff
@@ -837,8 +837,51 @@ def var(self, observable, wires, par):
         return var
 
     def sample(self, observable, wires, par, n=None):
-        raise NotImplementedError("Sampling is not supported in default.gaussian, "
-                                  "please install PennyLane-SF or another plugin capable of sampling")
+        """Return a sample of an observable.
+
+        .. note::
+
+            The ``default.gaussian`` plugin only supports sampling
+            from :class:`~.X`, :class:`~.P`, and :class:`~.QuadOperator`
+            observables.
+
+        Args:
+            observable (str): name of the observable
+            wires (Sequence[int]): subsystems the observable is to be measured on
+            par (tuple): parameters for the observable
+            n (int): Number of samples that should be obtained. Defaults to the
+                number of shots given as a parameter to the corresponding Device.
+
+        Returns:
+            array[float]: samples in an array of dimension ``(n, num_wires)``
+        """
+        if n is None:
+            n = self.shots
+
+        if n <= 0 or not isinstance(n, int):
+            raise ValueError("The number of samples must be a positive integer.")
+
+        if len(wires) != 1:
+            raise ValueError("Only one mode can be measured in homodyne.")
+
+        if observable == "X":
+            phi = 0.0
+        elif observable == "P":
+            phi = np.pi/2
+        elif observable == "QuadOperator":
+            phi = par[0]
+        else:
+            raise NotImplementedError("default.gaussian does not support sampling {}".format(observable))
+
+        mu, cov = self.reduced_state(wires)
+        rot = rotation(phi)
+
+        muphi = rot.T @ mu
+        covphi = rot.T @ cov @ rot
+
+        stdphi = np.sqrt(covphi[0, 0])
+        meanphi = muphi[0]
```

Comment:
I'm a bit confused by the implementation. It looks like the `wires` argument is not really being used? From what I'm reading, we use `wires` to determine the reduced state mean and covariance matrix. But then the samples are just obtained from the gaussian state of the first wire?

Reply:
```wires``` is only used to figure out which block of the covariance matrix and vector of means should be extracted in order to generate the normal samples

Reply:
Then maybe it's the documentation that is unclear? From reading, it seems like you should be able to submit a list of wires of arbitrary size (e.g., `wires=[0,1,3,6]`)

Reply:
It is confusing that wires is allowed to be a sequence of integers but at the same time we force that sequence to have length equal to 1. One reason to leave as is is that we might want to add the capability of doing homodyne for many modes in the future...

Reply:
Currently, operations/observables marked with class attribute `num_wires=1` only allow a list of wires of length 1.

This includes `qml.X`, `qml.P`, `qml.QuadOperator`. I don't see any reason to change this behaviour and have it accept a list of wires, as this isn't well defined (and we have already made the decision in an earlier issue to avoid implicit broadcasting of single-mode operations over a list of wires).